app/helpers/application_helper.rb

```diff
@@ -130,7 +130,11 @@ def present_from_to(from, to)
   end
 
   def govuk_date_range(from, to)
-    "#{from.to_date.to_formatted_s(:long)} to #{to.to_date.to_formatted_s(:long)}"
+    if to.present?
+      "#{from.to_date.to_formatted_s(:long)} to #{to.to_date.to_formatted_s(:long)}"
```

Comment:
just to check: do we need "From " at the start of this?

Reply:
Not for our current usage though appreciate that conflicts with how GovUK guidelines say we should be formatting dates